alf/networks/initializers.py

```diff
@@ -89,12 +94,14 @@ def variance_scaling_init(tensor,
     else:
         size = max(1.0, (fan_in + fan_out) / 2.0)
 
+    gain /= size
+
     if (calc_gain_after_activation and mode == "fan_in"
             and nonlinearity != "identity"):
         gain *= nn.init.calculate_gain(nonlinearity, nonlinearity_param)
 
-    std = gain / math.sqrt(size)
-    if distribution == "truncated_normal":
+    if distribution == "normal" or distribution == "truncated_normal":
+        std = math.sqrt(gain)
```

Comment:
This is not correct. TF's gain is square of PyTorch's gain. In `calculate_gain` the gain is after sqrt.

Reply:
Now converting to pytorch's convention both inside and outside.